tests/sysauth/sssd.pm

```diff
@@ -36,6 +36,9 @@ sub run {
     }
     script_run "zypper -n refresh && zypper -n in @test_subjects";
     script_run "cd; curl -L -v " . autoinst_url . "/data/sssd-tests > sssd-tests.data && cpio -id < sssd-tests.data && mv data sssd && ls sssd";
+    # save and upload sssd-tests.data
```

Comment:
I wonder if this actually makes sense, this data is already present in the data/directory of our test repo